src/pages/trading-specification/components/_trading_spec_table.tsx

```diff
@@ -0,0 +1,236 @@
+import React, { useEffect, useState } from 'react'
+import {
+    flexRender,
+    getCoreRowModel,
+    getSortedRowModel,
+    useReactTable,
+    getPaginationRowModel,
+} from '@tanstack/react-table'
+import type { SortingState } from '@tanstack/react-table'
+import styled from 'styled-components'
+import { TAvailableLiveMarkets } from '../_types'
+import useLiveColumns from '../_use-trading-spec-columns'
+import forex_specification from '../data/_forex_specification'
+import { market_specification } from './_constants'
+import {
+    TableData,
+    TableContainer,
+    TableRow,
+    StyledButton,
+    StyledButtonPage,
+    StyledPaginationContainer,
+} from './_elements'
+import AvailablePlatform from './_available-platform'
+import RightChevron from 'images/svg/trading-specification/right-chevron.svg'
+import LeftChevron from 'images/svg/trading-specification/left-chevron.svg'
+import SearchIcon from 'images/svg/help/search.svg'
+import { Flex } from 'components/containers'
+import { Localize } from 'components/localization'
+import { Header } from 'components/elements'
+import { sanitize } from 'common/utility'
+import useRegion from 'components/hooks/use-region'
+import device from 'themes/device'
+
+export type TLiveMarketTableProps = {
+    market: TAvailableLiveMarkets
+}
+const StyledFlex = styled(Flex)`
+    flex-direction: column;
+    gap: 20px;
+    padding: 5rem 0;
+
+    @media ${device.tabletL} {
+        width: 95%;
+    }
+`
+const SearchForm = styled.form`
+    position: relative;
+    align-items: center;
+    padding: 6px 8px;
+    width: 464px;
+    height: 32px;
+    border: 1px solid #d6dadb;
```

Comment:
please check it as well, we have to use only standard colors. if in design different, we have to ask designers about it or change standard colors